tests/GatewayTest.php

```diff
@@ -9,45 +11,47 @@
 use Omnipay\Common\CreditCard;
 use Omnipay\Tests\GatewayTestCase;
 use Omnipay\WestpacPaywayRest\Gateway;
+use Omnipay\WestpacPaywayRest\Message\CreateSingleUseCardTokenRequest;
+use Omnipay\WestpacPaywayRest\Message\PurchaseRequest;
 
 /**
  * @property Gateway gateway
  */
 class GatewayTest extends GatewayTestCase
 {
-    public function setUp()
+    public function setUp(): void
     {
         parent::setUp();
 
         $this->gateway = new Gateway($this->getHttpClient(), $this->getHttpRequest());
```

Comment:
`gateway` should be declared as a property in the class.

Reply:
It's declared in the parent class